octue/utils/colour.py

```diff
@@ -0,0 +1,25 @@
+import binascii
+
+
+WHITE = "ffffff"
+
+
+def round_color(h):
+    quot = 255 / 5
+    return round(h / quot)
+
+
+def ansi(rgb):
+    if isinstance(rgb, str):
+        rgb = binascii.a2b_hex(rgb)
+    if isinstance(rgb, bytes):
+        rgb = list(map(int, rgb))
+    red, green, blue = list(map(round_color, rgb))
+    return 16 + 36 * red + 6 * green + blue
+
+
+def colourise(s, foreground=WHITE, background=None):
```

Comment:
file name, and function names, conflict between US and UK spellings of colour/color. Can you make it consistent (I prefer the americanisation because that's what most of matlab, matplotlib, etc etc use so it comes more naturally to scientists in this context)

Reply:
Yep this is just what I copied from that other repo, haven't refactored it yet

Reply:
We've used UK spelling for other words in the SDK so I'm going to stick with that